src/hooks/useAbbreviatedNumber.ts

```diff
@@ -1,22 +1,29 @@
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
 
-export default function useAbbreviatedNumber(value: number) {
+export default function useAbbreviatedNumber() {
```

Comment:
I prefer if this receives the value and returns the value formatted.

But if you really need this to return the formatter I'd rename it to useAbbreviatedFormatter or similar... 🤔